product_specifications/DVjtom_67/readme.md

```diff
@@ -0,0 +1,149 @@
+
+## User Stories
+
+### CRITICAL
+- [ ] As an repository admin, I would like the tracking code snippet to be as easy to install as a google analytics tracker, So that I don't need to incur in additional technical infrastructure or developing resources.
+- [ ] As an repository admin, I would like the tracking code snippet not to incurr any GDPR breaches, So that I don't have to implement any consent policy.
+- [ ] As a bibliometrician, I would like the processing service to process usage in accordance to the COUNTER code of practice, So I can compare usage stats from using this method with other usage stats collected from other methods (SUSHI reports).
+- [ ] As a bibliometrician, I would like the collected usage stats were available via the Datacite GraphQL API, So that I can run my data analysis in jupyter notebooks and scripts.
+- [ ] As an repository admin, I would like to have access to the usage stats directly Fabrica, So I have a quick overview of the usage when adminitrating my DOIs.
+
+### IMPORTANT
+
+- [ ] As an repository admin, I would like the collected usage stats were available via the REST API, GraphQL and  data metrics badge, So that my usage stats can be distributed to the web.
+- [ ] As an repository admin, I would like to have access to a tracker ID directly from Fabrica, So I don't have to go to any other place to configure my usage stats processing.
+
+### NICE TO HAVE
+
+- [ ] As an repository admin, I would like the collected usage stats were available as SUSHI Reports (JSON) in the DataCite API (/reports), So that we can audit the monthly usage stats if we need it to do it.
+
+### Detailed User Stories
+
+```cucumber
+
+Feature: Usage Stats Tracker Installation
+
+        Scenario: Installing the Usage stats tracker correctly
+            Given that the user has a Usage stats tracker CODE
+              And that the user has access to a source code of a repository with DOIs  
+              When the user includes the the Usage stats tracker CODE with the correct DOI name in the landing pages source code
+              And the repository is deployed to the web
+              And a third party visit the landing page with the tracking CODE correctly embeded.
+             Then the tracking CODE should trigger a call to the DataCite Usage Stats Processing Service
+              And the triggered call should include the all data needed for usage stats processing. 
+
+        Scenario: Installing the Usage stats tracker incorrectly
+            Given that the user has a Usage stats tracker CODE
+              And that the user has access to a source code of a repository with DOIs  
+              When the user includes the the Usage stats tracker CODE but either the tracking CODE or the DOI name are missing the repository source code
+             Then an Error Message in the developer tools should be displaying 
```

Comment:
Is this done by us or plausible?

Reply:
by "our" tracker. We have to modify not only the plausible hosted service but also the tracker.